### PR TITLE
fix: Add security context to pvc.

### PR DESCRIPTION
**Pull request by yogeshg**
Actually tested it this time.
Screen Shot 2023-08-23 at 2 43 32 PM

### CODE REVIEW

**Review by vanpelt — APPROVED** [review]
🤪

File: weave.tf [hunk 1]
```diff
@@ -136,10 +136,10 @@ resource "kubernetes_deployment" "weave" {
...
           }
         }
-        securityContext {
-          fsGroup = 0
-          runAsUser = 999
-          fsGroupChangePolicy = "OnRootMismatch"
+        security_context {
+          fs_group = 0
+          run_as_user = 999
+          fs_group_change_policy = "OnRootMismatch"
```
Comment (yogeshg) [inline weave.tf:142]:
```suggestion
          fs_group_change_policy = "OnRootMismatch"
          
```

File: weave.tf [hunk 2]
```diff
@@ -136,10 +136,11 @@ resource "kubernetes_deployment" "weave" {
...
+
```
Comment (yogeshg) [inline weave.tf:143]:
```suggestion
```

**Comment by jsbroks** [comment]
This PR is included in version 1.11.2 :tada:

**Pull request merged**